.github/workflows/build-manual-complete.yml

```diff
@@ -30,12 +30,12 @@ jobs:
         run: |
           cd docker
           docker build --build-arg QF_REF=${{ inputs.ref }} --no-cache -f Dockerfile.x86_64_complete -t qf-solochain-complete:${{ github.sha }} .
-          docker tag qf-solochain-complete:${{ github.sha }} qfnetwork/qf-solochain-complete:${{ github.sha }}
+          docker tag qf-solochain-complete:${{ github.sha }} theqfnetworkadmin/qf-solochain-complete:${{ github.sha }}
 
       - name: Push Docker Image to Docker Hub (signed)
         env:
           DOCKER_CONTENT_TRUST: "1"
           DOCKER_CONTENT_TRUST_ROOT_PASSPHRASE: ${{ secrets.DCT_ROOT_PASSPHRASE }}
           DOCKER_CONTENT_TRUST_REPOSITORY_PASSPHRASE: ${{ secrets.DCT_REPO_PASSPHRASE }}
         run: |
-          docker push qfnetwork/qf-solochain-complete:${{ github.sha }}
\ No newline at end of file
+          docker push theqfnetworkadmin/qf-solochain-complete:${{ github.sha }}
```

Comment:
`qf-solochain` is a code repository name. Let's update image name to match with the target binary name, i.e. `qf-node? See https://hub.docker.com/r/theqfnetwork/qf-node.